sync-core/src/main/java/com/cloudant/sync/datastore/BasicDatastore.java

```diff
@@ -107,6 +107,7 @@ public BasicDatastore(String dir, String name) throws SQLException, IOException
         queue.updateSchema(DatastoreConstants.getSchemaVersion3(), 3);
         queue.updateSchema(DatastoreConstants.getSchemaVersion4(), 4);
         queue.updateSchema(DatastoreConstants.getSchemaVersion5(), 5);
+        queue.updateSchema(DatastoreConstants.getSchemaVersion6(), 6);
```

Comment:
I think you will also need some mechanism to stop an old version of our library opening a newer database - see how CDTDatastore does this.